recipes/buildkite-test-collector/LICENSE

```diff
@@ -0,0 +1,7 @@
+Copyright 2022 Buildkite
```

Comment:
You can delete this file. The license is in the tarball.

Reply:
Thank you, deleted it 👍